src/Compilers/CSharp/Portable/Symbols/RefKindExtensions.cs

```diff
@@ -1,22 +1,17 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Diagnostics;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal static partial class RefKindExtensions
     {
-        public static SyntaxToken GetToken(this RefKind refKind)
+        public static bool IsManagedReference(this RefKind refKind)
         {
-            if (refKind == RefKind.Out)
-            {
-                return SyntaxFactory.Token(SyntaxKind.OutKeyword);
-            }
-            if (refKind == RefKind.Ref)
-            {
-                return SyntaxFactory.Token(SyntaxKind.RefKeyword);
-            }
-            return default(SyntaxToken);
+            Debug.Assert(refKind <= RefKind.RefReadOnly);
+
+            return refKind != RefKind.None;
         }
 
         public static RefKind GetRefKind(this SyntaxKind syntaxKind)
```

Comment:
This is used in couple places involving arguments (not parameters). `In` parameters do not have anything at the call-site, so we are ok here, but must be careful. 

---
In reply to: [103290244](https://github.com/dotnet/roslyn/pull/17382#discussion_r103290244) [](ancestors = 103290244)